.vscode/launch.json

```diff
@@ -0,0 +1,17 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "type": "pwa-node",
+            "request": "launch",
+            "name": "Launch Program",
+            "skipFiles": [
+                "<node_internals>/**"
+            ],
+            "program": "${workspaceFolder}\\src\\utils\\DashboardHelperFunctions.js"
+        }
+    ]
+}
```

Comment:
This should probably be left out of the repo.

Reply:
Hmm I'm not sure what this is? Is this just local tooling that somehow got leaked into a commit?

Reply:
This is not something that I created, I thought it to be something that leaked in since I updated main

Reply:
the .vscode folder is local workspace settings. That is probably a debug launch configuration that was generated automatically at some point.

This file should be removed... or better, yet, we should add the .vscode folder to .gitignore.